lib/models/chat_history.dart

```diff
@@ -59,6 +59,10 @@ Stream<DeltaEvent> _handleDocumentChange(Map<String, List<Emote>> emotes,
           deleted: false,
           channelId: data['channelId']);
 
+      if (model.isCommand) {
```

Comment:
This will trigger if anyone sends a command. Can we detect only outbound messages?

Reply:
Off my head I think it is possible to add a function `bool get isStreamer` to TwitchMessageModel. I will have to see how to implement the code though.

Reply:
It would be simpler to instead check right before the user sends a message: https://github.com/muxable/rtchat/blob/a9e57b0415b2abe4cf4d6c725707163b54f4188d/lib/components/channel_panel.dart#L202